bleach/html5lib_shim.py

```diff
@@ -12,26 +12,51 @@
 
 import six
 
-# filter out html5lib deprecation warnings to use bleach
-warnings.simplefilter("ignore", category=DeprecationWarning)
+# ignore html5lib deprecation warnings to use bleach; we are bleach
+# apply before we import submodules that import html5lib
+warnings.filterwarnings(
+    "ignore",
+    message="html5lib's sanitizer is deprecated",
+    category=DeprecationWarning,
+    module="bleach._vendor.html5lib",
```

Comment:
If you nix this `module=` bit here, then will this apply to the module and the Sanitizer `__init__`?

Reply:
no that still fails